backend/app/middleware/tenant.py

```
@@ -0,0 +1,106 @@
+
+            token = authorization.split(" ")[1]
+
+            # Get database session
+            async with get_db() as db:
+                auth_service = AuthService(db)
+                user = await auth_service.get_user_by_token(token)
+
+                if user and user.is_active:
+                    return user.tenant_id
+
+            return None
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)